samples/aui/auidemo.cpp

```diff
@@ -290,118 +290,118 @@ class SettingsPanel : public wxPanel
     {
         //wxBoxSizer* vert = new wxBoxSizer(wxVERTICAL);
 
-        //vert->Add(FromDIP(1), FromDIP(1), 1, wxEXPAND);
+        //vert->Add(FromDIP(1), FromDIP(1), wxSizerFlags(1).Expand());
```

Comment:
The use of `FromDIP()` for expandable spacers is actually quite useless... And using `Expand()` for spacers doesn't make much sense neither. I.e. I'd write this as

```suggestion
        //vert->Add(FromDIP(1), FromDIP(1), wxSizerFlags(1).Expand());
```
```suggestion
        //vert->AddStretchSpacer(1);
```

(and, of course, the same for all not commented out lines too).

Reply:
Got it.